nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/TablePlotGUI.java

```diff
@@ -103,54 +108,94 @@ public void showPlotStage() {
             } catch (NullPointerException npEmc1) {
                 Alert alert = new Alert(Alert.AlertType.ERROR);
                 alert.setContentText("Error: Fit must first be performed.");
                 alert.showAndWait();
                 return;
             }
             tableView.getColumns().
-                    addListener((ListChangeListener) (c -> {
-                        updateAxisChoices();
-                    }));
+                    addListener((ListChangeListener) (c -> updateAxisChoices()));
 
             ToolBar toolBar = new ToolBar();
             MenuButton fileMenu = new MenuButton("File");
 
             MenuItem exportSVGMenuItem = new MenuItem("Export SVG...");
             fileMenu.getItems().add(exportSVGMenuItem);
             exportSVGMenuItem.setOnAction(e -> exportSVGAction());
+            CheckBox showFitCheckBox = new CheckBox("Fitting");
+            showFitCheckBox.setSelected(false);
+            showFitCheckBox.setOnAction(e -> toggleFitPane(showFitCheckBox));
 
-            toolBar.getItems().addAll(fileMenu);
+            ToolBar toolBar2 = new ToolBar();
 
+            Button button = new Button("Fit");
+            button.setOnAction(e -> analyze());
+            equationChoice.getItems().addAll("ExpAB", "ExpABC", "Gaussian", "A<->B");
+            equationChoice.setValue("ExpABC");
+            toolBar2.getItems().addAll(equationChoice, button);
             HBox hBox = new HBox();
             HBox.setHgrow(hBox, Priority.ALWAYS);
             hBox.setMinWidth(600);
-            hBox.getChildren().addAll(typelabel, chartTypeChoice,
+            toolBar.getItems().addAll(fileMenu,typelabel, chartTypeChoice,
                     xlabel, xArrayChoice, ylabel, yArrayChoice);
+            ToolBarUtils.addFiller(toolBar, 20,2000);
+            toolBar.getItems().add(showFitCheckBox);
             hBox.setAlignment(Pos.CENTER);
 
             VBox vBox = new VBox();
             vBox.setMinWidth(600);
-            vBox.getChildren().addAll(toolBar, hBox);
+            vBox.getChildren().addAll(toolBar);
+            fitVBox = new VBox();
+            fitVBox.setMinWidth(200);
+            fitVBox.getChildren().addAll(toolBar2, parTable);
+
             chartPane = new XYChartPane();
             activeChart = chartPane.getChart();
             borderPane.setTop(vBox);
             borderPane.setCenter(chartPane);
+            borderPane.setRight(null);
             stage.setScene(stageScene);
+            setupTable();
         }
         updateAxisChoices();
         stage.show();
         stage.toFront();
         updatePlot();
     }
 
+    private void toggleFitPane(CheckBox showFitBox) {
+        if (showFitBox.isSelected()) {
+            borderPane.setRight(fitVBox);
+        } else {
+            borderPane.setRight(null);
+        }
+
+    }
+    void setupTable() {
+        TableColumn<ParItem, String> columnNameColumn = new TableColumn<>("Column");
```

Comment:
A<->B is s fit to a reaction reactant A to product B.  It simultaneously fits the change in reactant and product so there there is only one fit result that is rendered as two lines

Reply:
Hmm so it is assumed then that the increasing line will be the product and the decreasing line is the reactant, so it can be interpreted easily, just the chart legend looks a little funny to have them both the same colour
![image](https://user-images.githubusercontent.com/64663455/202247145-f42fdfc3-b2bd-466e-98ae-f7ed53cd5fde.png)